server/apps/player/src/character.erl

```diff
@@ -1,179 +1,169 @@
 -module(character).
 
--export([create/2, player_characters/2, player_character/2, update/2, retrieve_near_players/2, activate/2, deactivate/4, harvest_resource/2]).
+-export([create/2, player_characters/2, player_character/2, update/2,
+         retrieve_near_players/2, activate/2, deactivate/4, harvest_resource/2]).
+
 -compile({parse_transform, do}).
 
-create(#{name := Name, 
-	 username := Username, 
-	 email := Email,
-	 constitution := Constitution,
-	 wisdom := Wisdom,
-	 strength := Strength,
-	 endurance := Endurance,
-	 intelligence := Intelligence,
-	 faith := Faith}, Connection) ->
-    Query = "INSERT INTO character.view (name, email, username, constitution, wisdom, strength, endurance, intelligence, faith) \
-             VALUES ($1::TEXT, $2::TEXT, $3::TEXT, $4::SMALLINT, $5::SMALLINT, $6::SMALLINT, $7::SMALLINT, $8::SMALLINT, $9::SMALLINT)",
-    Result = epgsql:equery(Connection, Query, [Name, Username, Email, Constitution, Wisdom, Strength, Endurance, Intelligence, Faith]),
-    do([postgres_m || 
-	   _ <- {Result, insert},
-	   ok]).
+create(#{name := Name,
+         username := Username,
+         email := Email,
+         constitution := Constitution,
+         wisdom := Wisdom,
+         strength := Strength,
+         endurance := Endurance,
+         intelligence := Intelligence,
+         faith := Faith},
+       Connection) ->
+    Query = database_queries:fetch_query("character", "create_character.sql"),
```

Comment:
I find the use case of .sql files to contain the queries a sus choice. Now you have to maintain a tiny library to fetch queries. Now the queries live far away from their actual use. And all of it to gain what it seems to be only highlighting of the SQL code. I don't know, seems like not a good trade to me. I probably would have voted against this in the past, but this is not the worst.

Reply:
I had to do it because not having SQL linting sucks, and also makes using `erlfmt` impossible (the formatter will rework the query to be unreadable).

Reply:
This and the following comment make me wonder if we should either better configure `erlfmt` (maybe via a config file?) or drop it entirely. This code will be maintained by people; not by for formatters and linters.

Reply:
The rest of the code is already formatted with `erlfmt`, it was at some point included in the [treefmt.nix](https://github.com/Dr-Nekoma/lyceum/blob/master/treefmt.nix) (I removed it, given that `nix fmt` would also mess up these two .erl files), we just never did it on `map.erl` and `character.erl` because if fucked up the queries big time.

Reply:
I find the tradeoff of having to maintain a tiny library for queries + queries far from actual use to make the formatter happier does not seem worth it to me at all. We are making developers' life worse to allow an external non-mandatory formatter happier. I will approve the PR given the speed we need, but we should revisit this later.